plugin/src/main/scala/org/virtuslab/akkasaferserializer/SerializerTrait.scala

```diff
@@ -0,0 +1,3 @@
+package org.virtuslab.akkasaferserializer
+
+class SerializerTrait extends scala.annotation.StaticAnnotation
```

Comment:
Actually... the name `SerializerTrait` is misleading a bit, since `Serializer` suggests a class like `akka.serialization.Serializer` that actively does serialize some 3rd party object... while this trait is really used to mark **serializable objects**.

Reply:
So... maybe `Serializability(Marker?)Trait`?